app/suggestion/forms.py

```diff
@@ -1,22 +1,37 @@
 from flask.ext.wtf import Form
-from wtforms.fields import SubmitField, TextField
+from wtforms.fields import FloatField, StringField, SubmitField
 from wtforms.validators import Email, InputRequired, Length
 
 
 class SuggestionForm(Form):
```

Comment:
Even though SuggestionForm (now ResourceSuggestionForm) and ResourceForm are conceptually related, I think code duplication is better than trying to set up from inheritance here, especially since the forms live in separate blueprints.